deposit/src/main/java/edu/unc/lib/deposit/normalize/Simple2N3BagJob.java

```diff
@@ -92,12 +81,12 @@ public void runJob() {
 		String slug = depositStatus.get(DepositField.depositSlug.name());
 		String mimetype = depositStatus.get(DepositField.fileMimetype.name());
 
-		String contentModel = depositStatus.get(hasModel.toString());
+		String contentModel = depositStatus.get(RDF.type.toString());
```

Comment:
I'm pretty sure this isn't going to work right out of the box.  Rather than try to resolve it, I suggest removing this and getting rid of all the code related to populateContainer, just assume its always creating a FileObject.  I'm pretty certain that populateContainer is dead code because this isn't actually how we create new folders or collections presently (it doesn't go through the deposit pipeline)